doc/DOCUMENTATION.org

```diff
@@ -421,7 +421,7 @@ contribution guidelines in [[./CONTRIBUTE.org][CONTRIBUTE.org]].
 
 ** Example: Themes Megapack example
 This is a simple =contrib= configuration layer listing a bunch of themes, you
```

Comment:
Yeah, I get it. Just having it as "`contrib` layer" instead of 'contribution layer' makes `contrib` thing very special. That's why I asked. Sorry for not being clear from the beginning :smile: